### PR TITLE
WIP: Issue #4753-refactor-services-declaration

### PR DESCRIPTION
**Pull request by dogeTheCoder**
#### Note this is still a work in progress i just have some questions.
refactors issue 4753: https://github.com/open-learning-exchange/planet/issues/4753
should i remove the whole providers attribute?
If so what should i do about MatIconRegistry?

### CODE REVIEW

**Comment by paulbert** [comment]
Nothing needs to happen with **MatIconRegistry**

**Comment by dogeTheCoder** [comment]
> PouchAuthService

I've removed the entire provider array and fixed the both PouchService's

File: src/app/shared/couchdb.service.ts [hunk 1]
```diff
@@ -9,7 +9,11 @@ import { findDocuments, inSelector } from './mangoQueries';
 
 class DatePlaceholder {}
 
-@Injectable()
+@Injectable({
+  providedIn: 'root'
+})
+
```
Comment (paulbert) [inline src/app/shared/couchdb.service.ts]: Please remove the extra added lines here

Comment (dogeTheCoder) [reply src/app/shared/couchdb.service.ts]: @paulbert just finished removing extra line and the unused imports

**Pull request merged**